assets/javascripts/src/reducers/gaTracking.es6

```diff
@@ -0,0 +1,12 @@
+import { TRACKING } from 'src/actions';
+
+const initialState = [false, false, false, false];
```

Comment:
There's one boolean per step in our checkout process. This is use to debounce tracking events.